### PR TITLE
Add job: periodic-github-ldap-user-group-creator

### PR DESCRIPTION
**Pull request by hongkailiu**
/cc @openshift/test-platform 

https://issues.redhat.com/browse/DPTP-1482

### CODE REVIEW

**Comment by hongkailiu** [comment]
/hold

wait for https://github.com/openshift/release/pull/22408/files

File: core-services/sanitize-prow-jobs/_config.yaml [hunk 1]
```diff
@@ -4739,6 +4739,7 @@ groups:
     - branch-ci-openshift-release-master-release-controller-annotate
     - branch-ci-openshift-release-master-vsphere-apply
     - openshift-release-master-config-bootstrapper
+    - periodic-github-ldap-user-group-creator
```
Comment (petr-muller) [inline core-services/sanitize-prow-jobs/_config.yaml:4742]: Shouldn't be needed after https://github.com/openshift/release/pull/22198

File: ci-operator/jobs/infra-periodics.yaml [hunk 2]
```diff
@@ -2129,3 +2129,46 @@ periodics:
...
+  decorate: true
+  decoration_config:
+    gcs_configuration:
+      bucket: origin-ci-private
+      job_url_prefix: https://deck-internal-ci.apps.ci.l2s4.p1.openshiftapps.com/view/
+      mediaTypes:
+        log: text/plain
+    gcs_credentials_secret: gce-sa-credentials-gcs-private
+  hidden: true
+  interval: 10m
```
Comment (petr-muller) [inline ci-operator/jobs/infra-periodics.yaml:2143]: Looks a bit excessive... I think a 12h period would be fine? Is it a problem if this has a day of latency?

Comment (droslean) [reply ci-operator/jobs/infra-periodics.yaml:2143]: I agree with Petr. How often the LDAP relations will change. I would be fine to run the job even once per day.

Comment (hongkailiu) [reply ci-operator/jobs/infra-periodics.yaml]: Changed to 24h.

File: core-services/sanitize-prow-jobs/_config.yaml (hunk 1 above)
Comment (hongkailiu) [reply core-services/sanitize-prow-jobs/_config.yaml]: Removed.

**Comment by hongkailiu** [comment]
Let me rehearse a bit.

**Comment by hongkailiu** [comment]
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/22409/pull-ci-openshift-release-master-pj-rehearse/1443920349631614976

> hidden jobs are not allowed to be rehearsed

**Pull request merged**